### DISCUSSION TITLE
Can we set up future seasons?

### DISCUSSION THREAD

**Discussion by holty07** [lardbit/nefarious#196] [Q&A]
If I have already seen all episodes released of a TV show, am I able to add that to my wanted list somehow for future seasons only?

**Comment by lardbit** [comment]
This is a good point.  nefarious currently only lets you watch future episodes of a show once you're watching at least 1 episode.

A workaround would simply to select a single episode that you've already seen, then the *Automatically watch future seasons* checkbox will appear.  Select that then you can deselect the episode you chose earlier.

I'll keep this discussion open until I can implement a solution.

![image](https://user-images.githubusercontent.com/45122868/173381835-b686321c-a41f-4997-960f-27fdcea4913c.png)
